scylla-cql/src/types/deserialize/row.rs

```diff
@@ -1325,3 +1354,47 @@ mod tests {
         }
     }
 }
+
+/// ```compile_fail
+///
+/// #[derive(scylla_macros::DeserializeRow)]
+/// #[scylla(crate = scylla_cql, skip_name_checks)]
+/// struct TestRow {}
+/// ```
+fn _test_struct_deserialization_name_check_skip_requires_enforce_order() {}
+
```

Comment:
Is the only purpose of those empty test functions describing what the test does? If so, would a comment be better?

Reply:
Hmm, the purpose of those functions is that they make the associated doctests run by the test runner. And their names are descriptive so that if a test fails, we instantly see, looking at its name, what feature has a fault.

Reply:
Ok, makes sense.